django/db/migrations/operations/models.py

```diff
@@ -42,8 +42,38 @@ def __init__(self, name, fields, options=None, bases=None, managers=None):
         self.options = options or {}
         self.bases = bases or (models.Model,)
         self.managers = managers or []
+
         super(CreateModel, self).__init__(name)
 
+        # Sanity-check that there are no duplicated field names, bases, or
+        # manager names
+        used_field_names = set()
+        for name, _ in fields:
+            if name in used_field_names:
+                raise ValueError(
+                    'You cannot have more than one field with the same name. '
```

Comment:
How about more simply: "Found duplicate <field/base/manager> '%s' in CreateModel operation."
Maybe you could create a helper function so we don't have to repeat a similar loop 3 times.